src/t8_cmesh/t8_cmesh_examples.c

```diff
@@ -940,21 +942,28 @@ t8_cmesh_set_vertices_2D (t8_cmesh_t cmesh, const t8_eclass_t eclass, const doub
   for (t8_locidx_t quad_y_id = 0; quad_y_id < quads_y; quad_y_id++) {
     for (t8_locidx_t quad_x_id = 0; quad_x_id < quads_x; quad_x_id++) {
 
-      t8_vec_axy (box, vertices, 1.0);                /* Vertex 0 */
-      t8_vec_axpyz (box, box_dir, vertices + 6, 1.0); /* Vertex 2 */
+      if (use_axis_aligned_geom && eclass == T8_ECLASS_QUAD) {
```

Comment:
Should we abort if `use_axis_aligned_geom && eclass != T8_ECLASS_QUAD`?

Reply:
Resturctured the program a little and added this feature too.